### PR TITLE
Repeat the header text, like the Drake video

### PR DESCRIPTION
**Pull request by gabebw**
The video repeats "HOTLINE BLING" 6 times. (I counted.)

### CODE REVIEW

File: app/views/homes/show.html.erb [hunk 1]
```diff
@@ -1,18 +1,22 @@
-<h1><%= title %></h1>
+<% 6.times do %>
+  <h1><%= title %></h1>
```
Comment (gabebw) [inline app/views/homes/show.html.erb]: Ideally we'd have one h1 per page. But in this case, every h1 has the same text. So I think it's OK?

I welcome opinions.

**Comment by gabebw** [comment]
Ideally we'd have one h1 per page. But in this case, every h1 has the same text. So I think it's OK?

I welcome opinions.

**Comment by edwardloveall** [comment]
I suppose that that point it's not really fulfilling its role as a heading and could be a bunch of spans instead. How accessible is this webring?

**Comment by gabebw** [comment]
> How accessible is this webring?

Uh, pretty accessible I think. It's mostly a bunch of URLs.

> I suppose that that point it's not really fulfilling its role as a heading and could be a bunch of spans instead

Hmm, spans feel odd, because it is _a_ header, just not _the_ header. What about h2s?

**Comment by edwardloveall** [comment]
`h2`s are fine too. I was thinking spans they don't denote any sort of heading. I think there's a way to hack extra text with css...

I'm hilariously imagining a screenreader reading "Hotline webring hotlinewebringhotlinewebringhotlinewebringhotlinewebringhotlinewebring"

**Comment by gabebw** [comment]
Done, with h2s: 46b239864bc85953c622bffdbc816cfc56b2619f

**Comment by edwardloveall** [comment]
:shipit:

**Pull request merged**